plugins/modules/storage/netapp/na_ontap_gather_facts.py

```diff
@@ -19,7 +19,7 @@
   alternative: Use M(na_ontap_info) instead.
 author: Piotr Olczak (@dprts) <polczak@redhat.com>
 extends_documentation_fragment:
-- netapp.ontap.netapp.na_ontap
+- community.general._netapp.na_ontap
```

Comment:
This module has been renamed to na_ontap_info (by RedHat).  How would en d user know that this module is deprecated.

Reply:
Because of the deprecation warning printed whenever it is used.